emmet-core/emmet/core/qchem/calculation.py

```diff
@@ -0,0 +1,437 @@
+"""Core definitions of a QChem calculation document."""
+
+# mypy: ignore-errors
+
+import logging
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+from pydantic import BaseModel, Field
+from pydantic.datetime_parse import datetime
+from pymatgen.io.qchem.inputs import QCInput
+from pymatgen.io.qchem.outputs import QCOutput
+from collections import OrderedDict
+import re
+
+from emmet.core.qchem.calc_types import (
+    LevelOfTheory,
+    CalcType,
+    TaskType,
+    level_of_theory,
+    calc_type,
+    task_type,
+)
+
+from emmet.core.qchem.task import QChemStatus
+
+__author__ = "Rishabh D. Guha <rdguha@lbl.gov>"
+logger = logging.getLogger(__name__)
+
+# class QChemObject(ValueEnum):
+# Not sure but can we have something like GRAD and HESS
+# as QChem data objects
```

Comment:
Maybe something we can discuss? Is the idea of this enum that you would have all of the different possible output files as possible enum values?